src/src/com/microsoft/aad/adal/CacheKey.java

```diff
@@ -155,6 +155,14 @@ public static String createMultiResourceRefreshTokenKey(AuthenticationRequest it
         return createCacheKey(item.getAuthority(), item.getResource(), item.getClientId(), true,
                 cacheUserId);
     }
+    
+    public static String createFamilyRefreshTokenKey(final AuthenticationRequest authRequest, final String userId) {
+        if (authRequest == null) {
+            throw new IllegalArgumentException("authentication request is null");
+        }
+        
+        return createCacheKey(authRequest.getAuthority(), null, null, true, userId);
```

Comment:
Actually if server returns value 1 (not Boolean true), for them it may already identify the family, then they can extend it to any number of families without any breakage.